libstorage/drivers/storage/rbd/utils/utils.go

```diff
@@ -177,6 +175,34 @@ func GetMappedRBDs(ctx types.Context) (map[string]string, error) {
 	return devMap, nil
 }
 
+//ParseMappedRBDs is a two-stage parse attempt because Ceph v13 has different JSON format to previous versions
+func ParseMappedRBDs(output *[]byte) (map[string]*rbdMappedEntry, error) {
+	rbdMap := map[string]*rbdMappedEntry{}
+
+	err := json.Unmarshal(*output, &rbdMap)
+	if err != nil {
+		type v13RbdMappedEntry struct {
```

Comment:
I'd rather see this type defined at the type, right after the origijal `rbdMappedEntry` struct.

I might suggest renaming the old struct to `rbdMappedEntryPreMimic` to denote it's the "old" version, but that's your call. no biggie on that one.